### ISSUE TITLE
ImportError: cannot import name 'MutableSequence'

### ISSUE THREAD

**Issue by AmazingMilesMorales** [Montvydas/translatesubs#7]
I ran 'pip install translatesubs' on my Windows 10 machine and when running 'translatesubs -h', I got the following order:

```
Traceback (most recent call last):
  File "C:\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Python310\Scripts\translatesubs.exe\__main__.py", line 4, in <module>
  File "C:\Python310\lib\site-packages\translatesubs\translatesubs.py", line 10, in <module>
    from .subs_manager import SubsManager
  File "C:\Python310\lib\site-packages\translatesubs\subs_manager.py", line 1, in <module>
    import pysubs2
  File "C:\Python310\lib\site-packages\pysubs2\__init__.py", line 1, in <module>
    from .ssafile import SSAFile
  File "C:\Python310\lib\site-packages\pysubs2\ssafile.py", line 1, in <module>
    from collections import MutableSequence
ImportError: cannot import name 'MutableSequence' from 'collections' (C:\Python310\lib\collections\__init__.py)
```

This occurred on both Python 3.8.x (don't remember which before I upgraded) and 3.10.7.

**Comment by Lanvaldear** [comment]
Edit ssafile.py and replace "from collections import MutableSequence" with "from collections.abc import MutableSequence".

**Comment by Montvydas** [comment]
Fixed at version 0.2.4.
